source/habitat_quality.rst

```diff
@@ -71,12 +71,11 @@ The impact of threats on habitat in a grid cell is mediated by four factors.
 	:label: (hq. 2)
 
 
-	
-where :math:`d_{xy}` is the linear distance between grid cells :math:`x` and :math:`y` and :math:`d_{r\ \mathrm{max}}` is the maximum effective distance of threat :math:`r`'s reach across space. Figure 1 illustrates the relationship between the distance-decay rate for a threat based on the maximum effective distance of the threat (linear and exponential). For example, if the user selects an exponential decline and the maximum impact distance of a threat is set at 1 km, the impact of the threat on a grid cell's habitat will decline by ~ 50% when the grid cell is 200 m from :math:`r`'s source. If :math:`i_{rxy} > 0` then grid cell :math:`x` is in degradation source :math:`ry`'s disturbance zone. (If the exponential function is used to describe the impact of degradation source :math:`r` on the landscape then the model ignores values of :math:`i_{rxy}` that are very close to 0 in order to expedite the modeling process.) To reiterate, if we have assigned species group-specific habitat suitability scores to each LULC then threat impact over space should be specific to the modeled species group.
+where :math:`d_{xy}` is the linear distance between grid cells :math:`x` and :math:`y` and :math:`d_{r\ \mathrm{max}}` is the maximum effective distance of threat :math:`r`'s reach across space. For exponential decay, the constant of 2.99 reduces the impact of the threat by 95% (to 5%) at the specified maximum effective distance. This is based on the traditional 95% cutoff used in statistics. The threats impact beyond the maximum effective distance is set 0 by the model. Figure 1 illustrates the exponential relationship between the distance-decay rate for a threat based on the maximum effective distance of the threat. For example, if the user selects an exponential decline and the maximum impact distance of a threat is set at 1 km, the impact of the threat on a grid cell's habitat will decline by ~ 55% when the grid cell is 200 m from :math:`r`'s source. If :math:`i_{rxy} > 0` then grid cell :math:`x` is in degradation source :math:`ry`'s disturbance zone. (If the exponential function is used to describe the impact of degradation source :math:`r` on the landscape then the model ignores values of :math:`i_{rxy}` that are very close to 0 in order to expedite the modeling process.) To reiterate, if we have assigned species group-specific habitat suitability scores to each LULC then threat impact over space should be specific to the modeled species group.
```

Comment:
This looks great! I only see two v. minor typos: there should be an apostrophe such that the sentence that starts "The threats impact..." should read "The threat's impact..." and then later in that sentence a missing work "to" "set **to** 0 in the model." I'm sorry I'm not sure how to edit directly, otherwise I would have!

Reply:
Thanks Lisa, this is totally fine! There is a way to "suggest a change" by selecting the "+" on a line to comment and then clicking on the "+/-" page looking icon to the right by the other formatting options. Thanks for engaging here!